conf/janus.transport.websockets.cfg.sample.in

```diff
@@ -0,0 +1,23 @@
+; WebSockets stuff: whether they should be enabled, which ports they
+; should use, and so on.
+[general]
+ws = yes					; Whether to enable the WebSockets interface
+ws_port = 8188				; WebSockets server port
+ws_ssl = no					; Whether to enable secure WebSockets
```

Comment:
Unless you meant renaming the secure WebSockets stuff to wss and keep ws for the unencrypted version?